pkgs/by-name/vs/vscode-runner/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDartApplication,
+  sqlite,
+  kdePackages,
+}:
+
+let
+  pname = "vscode-runner";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "Merrit";
+    repo = pname;
```

```suggestion
    repo = "vscode-runner";
```
see pname tracking issue